view/template/acquisition/yt2.php

```diff
@@ -66,20 +45,32 @@
 </section>
 <section class="claim section">
   <div class="inner">
+      <?php
+      // Change to something more meaningful
+      if (isset($_GET['error'])): echo "<div>error</div>";
+      endif;?>
     <div class="content">
       <div class="zigzag"></div>
       <h1>Own your identity. For real this time.</h1>
-      <form>
-        <div class="form-inner">
+      <form id="youtube_claim" method="post" action="/youtube/token/">
```

Comment:
This should be a 2 step form, not a form with 3 fields. The first step is just channel name. The second step is URL and email.